deepspeed/ops/lamb/lamb_cpu.py

```diff
@@ -0,0 +1,189 @@
+'''
```

Comment:
Is this a CPU implementation of the LAMB optimizer? It should be a separate PR, right?

Reply:
Yes I am sorry, I made a small mistake while creating PR so it got merged with the existing PR. I will create a new PR for this one.

Reply:
Got it. No worries. The cpu LAMB optimizer looks very interesting. I look forward to seeing the PR. Also could share the motivation and your experience with it in the PR? Thanks!